qa/L0_custom_image_preprocess/preprocess_test.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
```

Comment:
I see that this is a test just of the custom image preprocess backend. That is fine. I don't see why it also have "simple_ensemble_client" in the test. The should be a separate L0_ensemble_image_client test that runs the simple client that uses the ensemble model to perform image classification.